api/services/events/v1/doc.go

```diff
@@ -16,3 +16,8 @@
 
 // Package events defines the event pushing and subscription service.
 package events
+
+import types "github.com/containerd/containerd/api/types"
+
+// deprecated: Use github.com/containerd/containerd/api/types.Envelope
```

Comment:
I think this requires `Deprecated` to be capitalised;

```suggestion
// Deprecated: Use [types.Envelope].
```